.releaserc.yml

```diff
@@ -0,0 +1,6 @@
+plugins:
+  - "@semantic-release/commit-analyzer"
+  - "@semantic-release/release-notes-generator"
+  - "@semantic-release/github"
+  - "@semantic-release/npm"
+  - "@semantic-release/git"
```

Comment:
As per https://github.com/semantic-release/semantic-release/blob/master/docs/usage/plugins.md#plugins-declaration-and-execution-order

> If the plugins option is defined, it overrides the default plugin list, rather than merging with it.

So I had to list the defaults first 🤷‍♂️